### ISSUE TITLE
Evaluate relations for cities

### ISSUE THREAD

**Issue by max-dn** [der-stefan/OpenTopoMap#239]
For many place nodes, population is no longer entered. Instead, the node is entered as admin_centre of a relation and the important information is located there.

We should consider relations or at least copy the important information to the node...

For example Paris is currently missing on the map (node 17807753 in relation 7444).

**Comment by mboeringa** [comment]
@max-dn ,

Even if @joto's flex backend for osm2pgsql becomes a reality (https://github.com/openstreetmap/osm2pgsql/issues/1036), I think copying the "important information to the node" (the missing population tag in this particular case), will be a hard call.

Paris is currently a bit of a problem/nightmare in this respect, there are three(!) **type=boundary, boundary=administrative** relations defined with the name "Paris" for admin_levels 6, 7, and 8, that in this case (but this could well be, and is actually expected to be, different in other cases), even seem to contain the exact same relation members, so the outer boundary the same size and form. Just look at the references below.

Only the **admin_level=8** relation has the **population=x** tag. And that is in fact luck. What if all three relations had a different member set / size and correspondingly different population?

**Admin_level=6:**
https://www.openstreetmap.org/relation/71525

**Admin_level=7:**
https://www.openstreetmap.org/relation/1641193

**Admin_level=8:**
https://www.openstreetmap.org/relation/7444

By the way, I knew the tagging of OpenStreetMap was curious at times, but this new relation type made me laugh out loud:
https://www.openstreetmap.org/relation/3883620

_I certainly hope Facebook doesn't adopt the idea and add all his billion plus sign ups to OSM.._. ;-) That would surely take the OpenStreetMap servers down...

As to this specific problem. I re-added the population tag to the administrative center node. Don't know if it will last long, because I guess the local OSM member who removed it, will step in again.

Another (stop-gap) solution for country capitals is to check for the:
**capital=yes
admin_level=2**
tags, and preferentially label those even if no population present. This would at least work for most country capitals.

**Comment by max-dn** [comment]
Yes, the easier way is to assume some "default population" for place=city or capital=yes

**Comment by Juergeen** [comment]
Not sure wher there is a connection to ##286.
